### ISSUE TITLE
use module without only 

### ISSUE THREAD

**Issue by n0v0leg** [Kiborg1/composit-3D-model#3]
https://github.com/Kiborg1/composit-3D-model/blob/8ab895a58210c0f2908f4ecc1f60964ae8f39c69/Console1/Console1/Console1.f90#L4
`only` statement is omitted.
https://stackoverflow.com/questions/51686745/why-should-i-use-use-only-in-fortran

**Comment by n0v0leg** [comment]
https://stackoverflow.com/questions/51686745/why-should-i-use-use-only-in-fortran

**Comment by n0v0leg** [comment]
https://www.fortran90.org/src/best-practices.html#modules-and-programs

**Comment by n0v0leg** [comment]
https://stackoverflow.com/questions/51686745/why-should-i-use-use-only-in-fortran

**Comment by n0v0leg** [comment]
https://www.fortran.com/Fortran_Style.pdf 1.1.2:
> the default accessibility of each module should be private by including the private statement
